index.html

```diff
@@ -143,10 +143,49 @@ <h3>
           </li>
           <li>Run the following steps [=in parallel=]:
             <ol>
+              <li>Let |allPorts:list&lt;SerialPort&gt;| be an empty [=list=].
+              </li>
+              <li>[=list/For each=] available Bluetooth device:
+                <ol>
+                  <li>[=list/For each=] {{BluetoothServiceUUID}}
+                  |uuid:BluetoothServiceUUID| supported by the device:
+                    <ol>
+                      <li>If |uuid| is not a [=blocked Bluetooth service class
+                      UUID=]:
+                        <ol>
```

Comment:
I think this could be an unordered list, since it's listing "or"-conditions rather than a set of steps to execute.

Reply:
Done